pkg/services/store/entity/sqlstash/sql_storage_server.go

```diff
@@ -44,6 +45,7 @@ type sqlEntityServer struct {
 	dialect     migrator.Dialect
 	snowflake   *snowflake.Node
 	broadcaster Broadcaster[*entity.Entity]
+	ctx         context.Context
 }
 
 func (s *sqlEntityServer) Init() error {
```

Comment:
@DanCech where is `Init` called? i'm having trouble tracking it down

Reply:
Public methods like Read call it before they do any work, so basically it get lazily initialized on the first request.